README.md

```diff
@@ -64,41 +67,62 @@ This input is used to `source setup.sh` for any installed ROS binaries (e.g. ins
 This action defines [an output variable](https://help.github.com/en/actions/building-actions/metadata-syntax-for-github-actions#outputs): `ros-workspace-directory-name`.
 It contains the path to the root of the ROS workspace assembled by the action.
 
-The variable value should be used to retrieve logs, binaries, etc. after the action completes.
+The variable value can be used to retrieve logs, binaries, etc. after the action completes.
 
 ## Usage
 
 See [`action.yml`](action.yml) to get the list of inputs supported by this action.
 
 [action-ros-ci-template](https://github.com/ros-tooling/action-ros-ci-template) offers a template for using `action-ros-ci`.
 
-### Build and run tests for your ROS 2 package
+### Core Usage Patterns
```

Comment:
Could you update the table of contents at the top of the README?

Reply:
Done